RecoParticleFlow/Configuration/python/HepMCCopy_cfi.py

```diff
@@ -1,3 +1,4 @@
 import FWCore.ParameterSet.Config as cms
 
-generator = cms.EDProducer("HepMCCopy")
+from RecoParticleFlow.Configuration.HepMCCopy import HepMCCopy
+generator = HepMCCopy()
```

Comment:
here and elsewhere, isn't the usual pattern to do `clone()` instead? @cms-sw/core-l2

Reply:
This is the new syntax in the framework and is equivalent (actually better) than clone. The only suggestion I have is to do

`from RecoParticleFlow.Configuration.modules import HepMCCopy`

Reply:
Out of curiosity, what is the difference (or main advantage) between using "from RecoParticleFlow.Configuration.modules import HepMCCopy" over using "from RecoParticleFlow.Configuration.HepMCCopy import HepMCCopy"

Reply:
If we (Core) decide to change the name of the auto-generated HepMCCopy.py file the `import <stuff>.modules` will always continue to work.

Reply:
Addressed in 7b347075fc302cd09d3a2c157957d2f57fb8eb63